spec/classes/sensu_redis_spec.rb

```diff
@@ -132,5 +132,14 @@
 
       it { should contain_file('/etc/sensu/conf.d/redis.json').with_ensure('absent') }
     end # purge configs
+
+    [true,false].each do |value|
+      context "with redis_tls specified as #{value}" do
+        let(:params) { { :redis_tls => value } }
+
+        it { should contain_sensu_redis_config('testhost.domain.com').with_tls(value) }
```

Comment:
If it is `false` doesn't the provider delete the tls key?

Reply:
Yes but this just tests the parameter getting passed to `sensu_redis_config` resource which is either `true` or `false`.  This doesn't test the provider behavior.